deploy-as-code/helm/environments/mgramseva-qa.yaml

```diff
@@ -230,6 +231,15 @@ ws-services:
   is-external-workflow-enabled: "true"
 ifix-reference-adapter:
   keycloak-credentials-clientid: "mgramseva-qa"
+  pspcl-ifix-event-receiver-name: "mgramseva-qa"
+  ifix-fiscal-event-search-time-interval-minutes: "11"
```

Comment:
Don't forget to update the values once the QA is complete. Also, inform the same to the QA team. Otherwise, the same values will go into production.

Reply:
sure